src/protocols/ethereum/views/TransactionDetails.vue

```diff
@@ -77,16 +107,23 @@ export default defineComponent({
         : TX_DIRECTION.received,
     );
 
+    const isEthTransaction = computed(
```

Comment:
We know this is an Ethereum transaction because this component is an ethereum specific component. Did you mean `isCoin` or something?

Reply:
I meant, `isEth` as in is ETH the symbol and not short for Ethereum
But it is indeed confusing so I'll change this